src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -72,15 +75,32 @@ export default class TranscriptArea {
     ulElement.class = 'mx-auto';
     transcriptContainer.appendChild(ulElement);
     transcriptLines.forEach((transcriptLine) => {
-      ulElement.appendChild(
-          TranscriptLineElement.createTranscriptLineElement(transcriptLine));
+      const transcriptLineElement =
+          TranscriptLineElement.createTranscriptLineElement(transcriptLine);
+      ulElement.appendChild(transcriptLineElement);
+      TranscriptArea.#transcriptLineMap.set(
+          transcriptLine.transcriptKey.id, transcriptLineElement);
     });
   }
 
+  /** Increments the indicator corresponding to `transcriptLineKey` by 1. */
+  static incrementCommentIndicatorAt(transcriptLineKeyId) {
```

Comment:
transcriptLineMap shouldn't be static and so incrementCommentIndicatorAt shouldn't be static either.

Reply:
gotcha, updated!